src/EditorFeatures/Core.Wpf/Suggestions/PreviewChanges/PreviewChangesCodeAction.cs

```diff
@@ -40,14 +41,20 @@ public PreviewChangesCodeAction(Workspace workspace, CodeAction originalCodeActi
                         return ImmutableArray<CodeActionOperation>.Empty;
                     }
 
+                    var previewResult = await _getPreviewResultAsync(cancellationToken).ConfigureAwait(true);
+                    if (previewResult?.ChangeSummary is not { } changeSummary)
+                    {
+                        return ImmutableArray<CodeActionOperation>.Empty;
+                    }
```

Comment:
can this happen?

Reply:
I'm not sure if this can happen. The condition existed previously, but it would hide the Preview button as opposed to the new behavior that shows an empty preview.